src/main/java/net/coderbot/terrestria/init/TerrestriaBlocks.java

```diff
@@ -177,214 +173,38 @@ public static void init() {
+		BASALT_GRASS_BLOCK = TerrestriaRegistry.registerBlock("basalt_grass_block", new TerraformGrassBlock(BASALT_DIRT, Block.Settings.copy(Blocks.GRASS_BLOCK)));
+		BASALT = TerrestriaRegistry.registerBlock("basalt", new Block(FabricBlockSettings.copy(Blocks.STONE).materialColor(MaterialColor.BLACK).build()));
+
+		INDIAN_PAINTBRUSH = TerrestriaRegistry.registerBlock("indian_paintbrush", new BasaltFlowerBlock(StatusEffects.SATURATION, 6, Block.Settings.copy(Blocks.POPPY)));
+		MONSTERAS = TerrestriaRegistry.registerBlock("monsteras", new BasaltFlowerBlock(StatusEffects.REGENERATION, 2, Block.Settings.copy(Blocks.POPPY)));
+
+		POTTED_INDIAN_PAINTBRUSH = TerrestriaRegistry.registerBlock("potted_indian_paintbrush", new FlowerPotBlock(INDIAN_PAINTBRUSH, Block.Settings.copy(Blocks.POTTED_POPPY)));
+		POTTED_MONSTERAS = TerrestriaRegistry.registerBlock("potted_monsteras", new FlowerPotBlock(MONSTERAS, Block.Settings.copy(Blocks.POTTED_POPPY)));
+
+		POTTED_REDWOOD_SAPLING = TerrestriaRegistry.registerBlock("potted_redwood_sapling", new FlowerPotBlock(REDWOOD_SAPLING, Block.Settings.copy(Blocks.POTTED_POPPY)));
+		POTTED_HEMLOCK_SAPLING = TerrestriaRegistry.registerBlock("potted_hemlock_sapling", new FlowerPotBlock(HEMLOCK_SAPLING, Block.Settings.copy(Blocks.POTTED_POPPY)));
+		POTTED_RUBBER_SAPLING = TerrestriaRegistry.registerBlock("potted_rubber_sapling", new FlowerPotBlock(RUBBER_SAPLING, Block.Settings.copy(Blocks.POTTED_POPPY)));
+		POTTED_CYPRESS_SAPLING = TerrestriaRegistry.registerBlock("potted_cypress_sapling", new FlowerPotBlock(CYPRESS_SAPLING, Block.Settings.copy(Blocks.POTTED_POPPY)));
+		POTTED_WILLOW_SAPLING = TerrestriaRegistry.registerBlock("potted_willow_sapling", new FlowerPotBlock(WILLOW_SAPLING, Block.Settings.copy(Blocks.POTTED_POPPY)));
+		POTTED_JAPANESE_MAPLE_SAPLING = TerrestriaRegistry.registerBlock("potted_japanese_maple_sapling", new FlowerPotBlock(JAPANESE_MAPLE_SAPLING, Block.Settings.copy(Blocks.POTTED_POPPY)));
+		POTTED_JAPANESE_MAPLE_SHRUB_SAPLING = TerrestriaRegistry.registerBlock("potted_japanese_maple_shrub_sapling", new FlowerPotBlock(JAPANESE_MAPLE_SHRUB_SAPLING, Block.Settings.copy(Blocks.POTTED_POPPY)));
+		POTTED_DARK_JAPANESE_MAPLE_SAPLING = TerrestriaRegistry.registerBlock("potted_dark_japanese_maple_sapling", new FlowerPotBlock(DARK_JAPANESE_MAPLE_SAPLING, Block.Settings.copy(Blocks.POTTED_POPPY)));
+		POTTED_RAINBOW_EUCALYPTUS_SAPLING = TerrestriaRegistry.registerBlock("potted_rainbow_eucalyptus_sapling", new FlowerPotBlock(RAINBOW_EUCALYPTUS_SAPLING, Block.Settings.copy(Blocks.POTTED_POPPY)));
+		POTTED_SAKURA_SAPLING = TerrestriaRegistry.registerBlock("potted_sakura_sapling", new FlowerPotBlock(SAKURA_SAPLING, Block.Settings.copy(Blocks.POTTED_POPPY)));
+		POTTED_PALM_SAPLING = TerrestriaRegistry.registerBlock("potted_palm_sapling", new FlowerPotBlock(PALM_SAPLING, Block.Settings.copy(Blocks.POTTED_POPPY)));
 		// TODO: Stripped Logs, Stripped Wood
```

Comment:
We should remove this lol

Reply:
palm saplings were already in the mod before the palm branch

Reply:
No, this refers to the TODO saying to add stripped wood and logs

Reply:
oh, that wasnt from me, but  I can remove it now if you want